### ISSUE TITLE
Docker build fails

### ISSUE THREAD

**Issue by andig** [volkszaehler/docker#7]
Fails due to missing mbstring extension. Should also check how to upgrade to php 7.3.

**Comment by tofuSCHNITZEL** [comment]
yes I noticed that it was not working for rpi docker image - I want to rework all of it anyway
